systemtests/service_test.go

```diff
@@ -900,6 +896,7 @@ func (s *systemtestSuite) createServices(c *C, numServices int, tenant string, s
 		services = append(services, service)
 	}
 
+	/* FIXME:
```

Comment:
Those are not my changes. I have not worked on serviceLb. Will check if they are necessary or to be fixed and fix them accordingly.